libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -432,20 +419,9 @@ def construct_store_handler(buildaction,
     """
     Construct a result handler for storing results in a database.
     """
-    assert buildaction.analyzer_type in supported_analyzers, \
-        'Analyzer types should have been checked already.'
-
-    if buildaction.analyzer_type == CLANG_SA:
-        res_handler = result_handler_plist_to_db.PlistToDB(
-            buildaction,
-            None,
-            run_id)
-
-    elif buildaction.analyzer_type == CLANG_TIDY:
-        res_handler = result_handler_clang_tidy.ClangTidyPlistToDB(
```

Comment:
Removing this distintiction here made `result_handler_clang_tidy.ClangTidyPlistToDB` an obsolete class. Is this intended?